<?xml version="1.0"?>
<launch>

    <!-- <arg name="offset_x" default="-0.05"/>
    <arg name="offset_y" default="-0.03"/>
    <arg name="offset_z" default="0.21"/> -->
    <arg name="offset_x" default="0"/>
    <arg name="offset_y" default="-0.045"/>
    <arg name="offset_z" default="0.25"/>

    <!-- yz_table_pbvs -->
    <node pkg="visual_servoing3" name="yz_table_pbvs" type="yz_table_pbvs" output="screen">
        <param name="y_max" value="0.3200016" />     <!-- 0.32*(1+0.000005) -->
        <param name="z_max" value="0.55000275" />    <!-- 0.55*(1+0.000005) -->
        <param name="offset_x" value="$(arg offset_x)" />
        <param name="offset_y" value="$(arg offset_y)" />
        <param name="offset_z" value="$(arg offset_z)" />
        <param name="error_thershold" value="0.005" />
        <param name="base_vel_scale" value="0.995" />
        <param name="yz_table_enable" value="false" />
    </node>


</launch>
